xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_flash_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_flash_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Visio XML",
+    "extensions": [
+      "vsdx"
+    ],
+    "mediaType": "application/vnd-ms-visio.drawing",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_pdf_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Visio",
+    "extensions": [
+      "vsd"
+    ],
+    "mediaType": "application/vnd-visio",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_pdf_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Windows Bitmap",
+    "extensions": [
+      "bmp"
+    ],
+    "mediaType": "image/bmp",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_bmp_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_bmp_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Graphics Interchange Format",
+    "extensions": [
+      "gif"
+    ],
+    "mediaType": "image/gif",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_gif_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_gif_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Joint Photographic Experts Group",
+    "extensions": [
+      "jpg",
+      "jpeg"
+    ],
+    "mediaType": "image/jpeg",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_jpg_Export"
+      },
+      "SPREADSHEET": {
+        "FilterName": "calc_jpg_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_jpg_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_jpg_Export"
+      },
+      "WEB": {
+        "FilterName": "writer_web_jpg_Export"
+      }
+    }
+  },
+  // New one
+  {
+    "name": "Portable Network Graphics",
+    "extensions": [
+      "png"
+    ],
+    "mediaType": "image/png",
+    "inputFamily": "DRAWING",
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_png_Export"
+      },
+      "SPREADSHEET": {
+        "FilterName": "calc_png_Export"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_png_Export"
+      },
+      "TEXT": {
+        "FilterName": "writer_png_Export"
+      },
+      "WEB": {
+        "FilterName": "writer_web_png_Export"
+      }
+    }
   },
   {
     "name": "Scalable Vector Graphics",
-    "extensions": ["svg"],
+    "extensions": [
+      "svg"
+    ],
     "mediaType": "image/svg+xml",
     "inputFamily": "DRAWING",
-    "storeProperties": {"DRAWING": {"FilterName": "draw_svg_Export"}}
+    "storeProperties": {
+      "DRAWING": {
+        "FilterName": "draw_svg_Export"
+      },
+      // New properties
```

Comment:
We only use to have "DRAWING" property for svg. Sounds like it could make sense to have the others too.

Reply:
Yes, but I don't think we need the "New properties" comment.

Reply:
Yes it's same than for other comments: I wasn't planning to keep them. Will remove it.